ndmg/graph/gen_graph.py

```diff
@@ -290,14 +296,12 @@ def save_graph_png(self, graphname):
         import matplotlib
         matplotlib.use('agg')
         from matplotlib import pyplot as plt
-        from nilearn.plotting import plot_matrix
+        from graspy.utils import ptr
+        from graspy.plot import heatmap
 
-        from sklearn.preprocessing import normalize
         conn_matrix = np.array(nx.to_numpy_matrix(self.g))
-        conn_matrix = normalize(conn_matrix)
-        [z_min, z_max] = -np.abs(conn_matrix).max(), np.abs(conn_matrix).max()
-        plot_matrix(conn_matrix, figure=(10, 10), vmax=z_max * 0.5, vmin=z_min * 0.5, auto_fit=True, grid=False,
-                    colorbar=False)
+        conn_matrix = ptr.pass_to_ranks(conn_matrix)
+        heatmap(conn_matrix)
```

Comment:
I think we still need to normalize plotting (e.g., vmin and vmax are 0 and 1, respectively)

Reply:
Actually, if you look through the graspy code for heatmap, you will see that vmin/vmax are inferred based on the data if these parameters are not set explicitly. @j1c can correct me if I'm wrong this.

Reply:
> Actually, if you look through the graspy code for heatmap, you will see that vmin/vmax are inferred based on the data if these parameters are not set explicitly. @j1c can correct me if I'm wrong this.

Yes, that's true - the reason we need to hardcore these parameters is so that we can compare multiple plots. If I have 2 heatmaps, and vmax is `1` on the first, and `10` on the second, and the underlying data is the same, they will look completely different.

Reply:
`vmax` and `vmin` just clips the data. if `vmax` is 1, then all values above 1 is set to 1. 

This is why i think it makes sense to at least PTR prior to plotting because then values are always between 0 and 1 so no need to mess with `vmax` and `vmin`, and makes all the small values visible.

Reply:
Also graspy heatmap function also allow you to just pass in a `transform` argument that will take care of PTR for you so no need to PTR separately before plotting.

Reply:
Thanks @j1c . Thought so!

Reply:
Ok, thanks dude